### ISSUE TITLE
Firefox 97 fails to install

### ISSUE THREAD

**Issue by ojii** [CircleCI-Public/browser-tools-orb#38]
### Orb version

1.2.4

### What happened

```
System detected as Linux
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   607  100   607    0     0   6897      0 --:--:-- --:--:-- --:--:--  6897
Latest stable version of Firefox is 97.0
gpg: Signature made Thu Feb  3 00:24:17 2022 UTC
gpg:                using RSA key 4360FE2109C49763186F8E21EBE41E90F6F12F6D
gpg: Good signature from "Mozilla Software Releases <release@mozilla.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 14F2 6682 D091 6CDD 81E3  7B6D 61B7 B526 D98F 0353
     Subkey fingerprint: 4360 FE21 09C4 9763 186F  8E21 EBE4 1E90 F6F1 2F6D
gpg: Signature made Wed Feb  2 20:05:18 2022 UTC
gpg:                using RSA key 4360FE2109C49763186F8E21EBE41E90F6F12F6D
gpg: Good signature from "Mozilla Software Releases <release@mozilla.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 14F2 6682 D091 6CDD 81E3  7B6D 61B7 B526 D98F 0353
     Subkey fingerprint: 4360 FE21 09C4 9763 186F  8E21 EBE4 1E90 F6F1 2F6D
Can't open SHA512SUMS: No such file or directory.
linux-x86_64-en-US-firefox-97.0.tar.bz2: OK
XPCOMGlueLoad error for file /usr/local/bin/firefox-97.0/libxul.so:
libasound.so.2: cannot open shared object file: No such file or directory
Couldn't load XPCOM.
Something went wrong; the specified version of Firefox could not be installed

Exited with code exit status 1
```

This is running on `python:3.9.7-bullseye`. before calling `browser-tools/install-firefox` the following commands are run:

```
apt-get update
apt-get -y install xmlsec1 gettext jq
apt-get -y upgrade openssl
```

### Expected behavior

Expect firefox to install normally.

**Comment by haydenmcp** [comment]
I'm seeing the same issue

**Comment by haydenmcp** [comment]
In case it helps, I tested my build by installing firefox v96.0 and the installation works. It'll allow you to move forward if you're stuck.

**Comment by ojii** [comment]
@haydenmcp we fixed it by apt installing `libasound-dev` before running the orb

**Comment by haydenmcp** [comment]
Thanks, that worked for me as well. :-)

**Comment by alcarasj** [comment]
+1 for this, experiecing the same issue. Build ref: https://app.circleci.com/pipelines/github/alcarasj/jerico-xyz/10585/workflows/70ae86a6-c5ba-4869-93e7-d7835ffb5681/jobs/10690

**Comment by haydenmcp** [comment]
@alcarasj Check out the fix @ojii mentioned. It worked for me as well.

**Comment by felicianotech** [comment]
Hey everyone. Apologies for the delay on this. We're actively working to improve this response time.

A fix has been made and is available in circleci/browser-tools@1.2.5.
